packages/server/src/migration.ts

```diff
@@ -0,0 +1,10 @@
+import * as migrateMongo from "migrate-mongo";
+import * as config from "../migrate-mongo-config.js";
+
+export async function migrate() {
+  // @ts-expect-error ts(2345)
+  migrateMongo.config.set(config);
+  const { db, client } = await migrateMongo.database.connect();
```

Comment:
For some reason, this produces the type error `Type '{ useNewUrlParser: boolean; useUnifiedTopology: boolean; }' has no properties in common with type 'MongoClientOptions'.ts(2345)`, but both of these properties were used by default (added by `migrate-mongo initialise`).